day_two_guide/topics/api_service_status.adoc

```diff
@@ -6,28 +6,22 @@ Module included in the following assemblies:
 * day_two_guide/environment_health_checks.adoc
 ////
 
-The OpenShift API service, `atomic-openshift-master-api.service`, runs on all
-master instances. To see the status of the service:
+The OpenShift API service runs on all
+master instances. To see the status of the service,
+view the pods in the *kube-system* project:
```

Comment:
There are three types of pods: api, controllers, and etcd in the 'kube-system' project:
If here focus API service, then you can say `view the master-api pods in the *kube-system* project` and use the command with a label to get master-api pod only as below:
` oc get pod -n kube-system -l openshift.io/component=api`